car.capnp

```diff
@@ -114,6 +114,7 @@ struct CarEvent @0x9b1657f34caf3ad3 {
     canBusMissing @111;
     controlsdLagging @112;
     resumeBlocked @113;
+    steerTimeLimit @116;
```

Comment:
Maybe you an re-use accFaultedTemp since we never really used it?

Reply:
Done.